### PR TITLE
Wire up ColonyRoleSet events with subgraph

### PR DESCRIPTION
**Pull request by ArmandoGraterol**
* Added handler for ColonyRoleSet to colony mapping, and yaml file
* Added boolean event value option to event mapping

### CODE REVIEW

**Review by chinins — COMMENTED** [review]
One thing to add to Alicja's & Raul's comments which I didn't see anybody mention.

I created a new team and then tried to change permissions for this new team. And I can't do it (please, see the gif). There was no error message shown, just little popover on hover that I don't have permission to do that.

2 points here:
- it seems wrong that having all of the permissions in the root domain and having created the team, I can't change any permissions and I don't have any permissions in this domain. Who would then be able to change any permissions? 
- if i can't change the permissions for a particular domain, there should be a clear permission error shown.

I am trying to click on each role but it's not clickable. Nothing else is disabled in the form:
permissions

**Comment by rdig** [comment]
@chinins 

> I created a new team and then tried to change permissions for this new team. And I can't do it (please, see the gif). There was no error message shown, just little popover on hover that I don't have permission to do that.

I don't think you should be able to change permissions for your own user, in a subdomain, since you already have all the permissions in that subdomain inherited from root.

That part is correct

However, you are right in the sense that for this particular case, the checkboxes should be disabled. @ArmandoGraterol make that validation happen.

File: src/mappings/event.ts [hunk 1]
```diff
@@ -26,6 +26,13 @@ export function handleEvent(eventName: String, event: ethereum.Event, associated
     } else if (event.parameters[i].value.kind == ethereum.ValueKind.FIXED_BYTES) {
       encoder.setString(event.parameters[i].name, event.parameters[i].value.toBytes().toHexString())
       log.info("{}", [event.parameters[i].value.toBytes().toHexString()])
+    } else if (event.parameters[i].value.kind == ethereum.ValueKind.BOOL) {
+      /* I feel dirty doing this. Suggestions are welcomed
+      String(value) isn't a valid operation, and value.toBoolean().toString() isn't valid also
+      (Even tho it's valid in vanilla Javascript) */
```
Comment (rdig) [inline src/mappings/event.ts:32]: That's because this is assembly script, not javascript

**Review by rdig — APPROVED** [review]
Good to go

**Pull request merged**